### PR TITLE
lncli: fix description of connect/disconnect peer

### PR DESCRIPTION
**Pull request by kilrau**
## Change Description
Fix description of `lncli connect` & `lncli disconnect`

## Steps to Test
`lncli connect --help` & `lncli disconnect --help`

### Testing
- [ ] Your PR passes all CI checks.
- [ ] Tests covering the positive and negative (error paths) are included.
- [ ] Bug fixes contain tests triggering the bug to prevent regressions.

### Code Style and Documentation
- [ ] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [ ] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [ ] Any new logging statements use an appropriate subsystem and logging level.
- [ ]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

**Review by guggero — APPROVED** [review]
Thanks, LGTM :tada:

**Review by dstadulis — APPROVED** [review]
Thanks improving the accuracy!

**Review by saubyk — APPROVED** [review]
Ack

File: cmd/lncli/commands.go [hunk 1]
```diff
@@ -682,7 +682,7 @@ func connectPeer(ctx *cli.Context) error {
 var disconnectCommand = cli.Command{
 	Name:      "disconnect",
 	Category:  "Peers",
-	Usage:     "Disconnect a remote lnd peer identified by public key.",
+	Usage:     "Disconnect a remote lightning peer identified by public key.",
```
Comment (guggero) [inline cmd/lncli/commands.go:685]: The linter is failing on a too long line now, need to break this string.

**Review by guggero — CHANGES_REQUESTED** [review]

**Comment by guggero** [comment]
Thanks. Can you please squash the commits into a single one, using the [prefix as described in the contribution guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#ideal-git-commit-structure), then I can merge.

**Pull request merged**